internal/server/v1/propertyvalues/state.go

```diff
@@ -220,18 +220,19 @@ func (s *outState) init(
 	return nil
 }
 
-func (s *state) getPagination() *pb.PaginationInfo {
-	res := &pb.PaginationInfo{}
+func (s *state) getPagination(direction string) *pb.PaginationInfo {
```

Comment:
Arg "direction" is not used in this function?

Reply:
Good catch! updated.